x-pack/plugins/alerting/server/lib/get_docs_for_rule_type_by_producer.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const getDocsForRuleTypeByProducer = (
```

Comment:
Using hardcoded links for now. Will replace it later with the docLinks service, when the support for the server usage will be implemented (opened [issue](https://github.com/elastic/kibana/issues/95389))